pages/common/doctl-apps.md

```diff
@@ -0,0 +1,40 @@
+# doctl-apps
+
+> Used to manage digitalocean apps.
+> More information: <https://docs.digitalocean.com/>.
+
+- Display commands for working with apps:
```

Comment:
Is the output of this command the help page?

Reply:
I think so?